packages/shorebird_cli/lib/src/commands/build/build_apk_command.dart

```diff
@@ -49,6 +49,13 @@ class BuildApkCommand extends ShorebirdCommand
       return ExitCode.noUser.code;
     }
 
+    if (!hasShorebirdYaml) {
+      logger.err(
```

Comment:
I wonder whether this should be another validation check? cc @bryanoltman

Reply:
I think that's a great idea

Reply:
Sure, Update check with validation check.